packages/gatsby-source-contentful/src/__tests__/normalize.js

```diff
@@ -85,12 +213,41 @@ describe(`Process contentful data (by name)`, () => {
         space,
         useNameForId: true,
         pluginConfig,
+        unstable_createNodeManifest,
       })
     })
-    expect(createNode.mock.calls).toMatchSnapshot()
 
-    // Relevant to compare to compare warm and cold situation. Actual number not relevant.
-    expect(createNode.mock.calls.length).toBe(74) // "cold build entries" count
+    const nodeTypeCounts = countCreatedNodeTypesFromMock(createNode.mock)
+
+    expect(Object.keys(nodeTypeCounts)).toHaveLength(15)
+
+    expect(nodeTypeCounts).toEqual(
+      expect.objectContaining({
+        // 3 Brand Contentful entries
+        ContentfulBrand: 6,
+        contentfulBrandCompanyDescriptionTextNode: 6,
+        contentfulBrandCompanyNameTextNode: 6,
+        // 2 Category Contentful entries
+        ContentfulCategory: 4,
+        contentfulCategoryCategoryDescriptionTextNode: 4,
+        contentfulCategoryTitleTextNode: 4,
+        ContentfulContentType: contentTypeItems.length,
+        // 1 JSON Test Contentful entry
+        ContentfulJsonTest: 2,
+        contentfulJsonTestJsonStringTestJsonNode: 2,
+        contentfulJsonTestJsonTestJsonNode: 2,
+        // 4 Product Contentful entries
+        ContentfulProduct: 8,
+        contentfulProductProductDescriptionTextNode: 8,
+        contentfulProductProductNameTextNode: 8,
+        // 1 Remark Test Contentful entry
+        ContentfulRemarkTest: 2,
+        contentfulRemarkTestContentTextNode: 2,
+      })
+    )
+
+    // Relevant to compare to compare warm and cold situation
+    expect(createNode.mock.calls.length).toBe(69) // "cold build entries" count
```

Comment:
69 is this intentional 😆

Reply:
especially in warm and cold situations lol